rulesdoc.md

```diff
@@ -538,15 +556,11 @@ During the adventuring day you will sometimes need to take a rest to recover hea
 
 ### Short Rest
 
-Short rests require an hour of uninterrupted rest to complete. During a short rest you may spend any number of healing surges to restore lost wounds. 
-
-Spellcasters restore up to 1/3rd of their level, rounded up, in spell slots. You may restore any number of spell slots this way as long as the sum level is not greater than 1/3rd of your level.
+Short rests require an hour of uninterrupted rest to complete. During a short rest you may spend any number of healing surges to restore lost wounds. You regain mana equal to half your character's level, rounded up, with a minimum of 1.
 
 ### Long Rest
 
-Long rests require 8 hours of rest, with at most 2 hours of non-strenuous activity, such as keeping watch, preparing spells, brewing potions, etc. You automatically recover 1 wound and 3 healing surges over a long rest, and you may spend any number of healing surges to restore wounds.
-
-Spellcasters restore up to their level in spell slots. You may restore any number of spell slots this way as long as the sum level is not greater than your level. In addition you may prepare new spells for the day.
+Long rests require 8 hours of rest, with at most 2 hours of non-strenuous activity, such as keeping watch, preparing spells, brewing potions, etc. You automatically recover 1 wound and 3 healing surges over a long rest, and you may spend any number of healing surges to restore wounds. You regain mana equal to 3 + your character's level.
 
```

Comment:
It seems odd that you would gain more mana back on short rest than long rest.  I don't see why you wouldn't just give them all their mana back.

At least with health, long rest gives you a resource you can't get back any other way (healing surge).  With mana, it's the same resource so you should get more back with long rest.  Alternatively you can make a "mana surge" or something and make it similar to wounds I suppose if you want to limit the resource in some way.

Reply:
you do get more back on long rest. On a short rest you get half your level, on a long rest you get 3 + your level. It's a similar recovery system to health where it can take more than a day to recover from very low mana/wounds.

Reply:
Oh, my bad.  For some reason I thought it was half the total mana pool on short test.

Reply:
Shouldn't the amount you recover be tied to Spell Power?

Reply:
Not exactly. It would be difficult to base it on spell power as mana is given based on reaching a certain threshold of spellpower. It'll be awkward to have you regain a fraction of your spell power (like calculating 1/6th per short rest or something) and I'm not sure how that'd scale.

right now it works somewhat similar to health regen. You always regen a fixed amount not tied to another stat. How many healing surges or mana you have just means you have more healing/spells available at max, but your regeneration is roughly the same.